warehouse/query-core/src/main/java/datawave/query/index/lookup/RangeStream.java

```diff
@@ -238,20 +238,18 @@ protected void shutdownThreads() {
                 }
                 if (queryStream.context() == StreamContext.VARIABLE) {
                     context = StreamContext.PRESENT;
-                } else
+                } else {
                     context = queryStream.context();
+                }
                 
                 if (log.isDebugEnabled()) {
                     log.debug("Query returned a stream with a context of " + this.context);
-                    if (queryStream != null) {
```

Comment:
[ERROR] Nullcheck of RangeStream.queryStream at line 253 of value previously dereferenced in datawave.query.index.lookup.RangeStream.iterator() [datawave.query.index.lookup.RangeStream, datawave.query.index.lookup.RangeStream, datawave.query.index.lookup.RangeStream] At RangeStream.java:[line 239]Redundant null check at RangeStream.java:[line 253]Another occurrence at RangeStream.java:[line 246] RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE